packages/lucide-figma/src/api/fetchIcons.ts

```diff
@@ -23,8 +23,8 @@ export const fetchIcons = async (cachedIcons? : LucideIcons): Promise<LucideIcon
     return cachedIcons
   }
 
-  const iconNodesResponse = await fetch(`https://unpkg.com/lucide-static@${packageJson.version}/icon-nodes.json`)
-  const tagsResponse = await fetch('https://unpkg.com/lucide-static@latest/tags.json')
+  const iconNodesResponse = await fetch(`https://lucide.dev/api/icon-nodes`)
+  const tagsResponse = await fetch('https://lucide.dev/api/tags')
```

Comment:
I would only add those endpoints when they are needed and keep using the CDN for now :man_shrugging:.
```suggestion
  const [iconNodesResponse, tagsResponse] = await Promise.all([
    fetch(`https://unpkg.com/lucide-static@${packageJson.version}/icon-nodes.json`),
    fetch('https://unpkg.com/lucide-static@latest/tags.json')
  ])
```

Reply:
> I would only add those endpoints when they are needed and keep using the CDN for now 🤷‍♂️.

We've discussed this with the other maintainers and we intend to move forward with deprecating the CDN call and switching to an API.

Reply:
You could still put it in a `Promise.all` :+1:

Reply:
Sure, that's a no brainer, thx.